src/outputs/parthenon_hdf5.cpp

```diff
@@ -310,13 +310,13 @@ void PHDF5Output::WriteOutputFile(Mesh *pm, ParameterInput *pin, SimTime *tm) {
       // open vector space
       // write mesh block size
       const hsize_t xDims[1] = {3};
-      int meshblock_size[xDims[0]] = {nx1, nx2, nx3};
+      int meshblock_size[3] = {nx1, nx2, nx3};
```

Comment:
Why specify 3 for the array length? We can just let the compiler figure it out
```suggestion
      int meshblock_size[] = {nx1, nx2, nx3};
```

Reply:
fixed